daemon/daemon_unix.go

```diff
@@ -596,6 +609,10 @@ func (daemon *Daemon) reloadPlatform(conf *config.Config, attributes map[string]
 		daemon.configStore.ShmSize = conf.ShmSize
 	}
 
+	if conf.IpcMode != "" {
```

Comment:
I fail to see the reason why a simple comparison is not enough, and what can be wrong if we use it instead of conf.IsValueSet(). From what I see, it makes sense to use it to distinguish the case when a parameter is not set from the case when a parameter is set to `nul`. Maybe I'm missing something?